src/sourmash/sourmash_args.py

```diff
@@ -156,6 +159,46 @@ def report_picklist(args, picklist):
             sys.exit(-1)
 
 
+def load_include_pattern(args):
```

Comment:
function name is a bit confusing bc this handles both inclusion and exclusion cli options (though I do see that yes, with either case, this function determines what to include).

Maybe `load_search_pattern`?  or compile/prepare search pattern?

Reply:
👍

Reply:
fixed in 88812335 -> `load_include_exclude_db_patterns`